PyInstaller/utils/win32/versioninfo.py

```diff
@@ -11,12 +11,15 @@
 
 import codecs
 import struct
+import sys
 
 import pywintypes
 import win32api
 # ::TODO:: #1920 revert to using pypi version
 from ...lib import pefile
 
+if sys.version_info >= (3, 5, 0):
+	pywintypes.Unicode = str
 
```

Comment:
Move this statement directly into compat.py so that it's executed as soon as compat is imported. Use `is_py3`.